tools/vf-component-library/src/site/components/index.njk

```diff
@@ -53,29 +74,28 @@ They're generated by the `vf-componenet-rollup`:
 {% endfor -%}
 
 <!-- some tools don't have a natural place in the component library and are linked to github -->
-<h3 class="vf-links__heading">Tools</h3>
-<ul class="vf-links__list vf-links__list--secondary | vf-list">
-  <li class="vf-list__item">
-    <a class="vf-list__link" href="https://github.com/visual-framework/vf-core/tree/develop/tools/vf-extensions">
+<h3 class="">Tools</h3>
```

Comment:
just spotted these classes have been remove.

Reply:
yeah, it's because that looked visually awkward, here's how it looks now: 

![image](https://user-images.githubusercontent.com/928100/114728376-52a55700-9d3f-11eb-9cee-e48e4ad54545.png)

I think the standard vf-content bullets are better, at least for now ... i could see a redesign of this page coming soonish. 

![image](https://user-images.githubusercontent.com/928100/114728469-694bae00-9d3f-11eb-8321-7e6d947f2966.png)

Reply:
Do we need the default `vf-list` though for styling or no? 

https://latest.visual-framework.dev//components/vf-list/index.html?q=list#vf-list--default

Reply:
I think the default (grey, no bullets) make the lists in this context hard to scan — but as I alluded to earlier, I think this page could use a bit of a rethink in how we're show "all the the things", so I think the list is coming up short, with or without bullets.

Reply:
Collecting my thoughts a little on this, I think "how others do it" show what I'm getting at. 

Little representative images are probably out-of-scope for the moment, but I think we can use the no-image cards to good effect. 

![image](https://user-images.githubusercontent.com/928100/114758371-2c8eaf80-9d5d-11eb-98a9-0308a7eeb68d.png)

![image](https://user-images.githubusercontent.com/928100/114758392-33b5bd80-9d5d-11eb-9e8d-4cb56d6df808.png)

![image](https://user-images.githubusercontent.com/928100/114758403-37e1db00-9d5d-11eb-83a0-65a76a211009.png)

Reply:
Something like this, but I dunno, doesn't feel quite right yet?

![image](https://user-images.githubusercontent.com/928100/114760121-49c47d80-9d5f-11eb-9f9d-c450ccb26e8f.png)

Reply:
maybe split patterns out completely … * coughs * live govuk 
![Screenshot 2021-04-15 at 08 41 17](https://user-images.githubusercontent.com/925197/114832502-765fb000-9dc6-11eb-8d26-916374abc5b0.png)